### ISSUE TITLE
NVDA does not announce the unavailability status of the windows10 2004 update

### ISSUE THREAD

**Issue by burakyuksek** [nvaccess/nvda#11507]
### Steps to reproduce:
1. Open windows update
2. Check for updates
3. If windows 10 version 2004 update is not available, windows will tell you that fact and narrator can see this. NVDA cannot do this any way, unless you physically focus the mouse to that text.
### Actual behavior:
NVDA does not read the text mentioned above
### Expected behavior:
NVDA reads the text above, either automatically when update check is finished, or it should somehow be accessible with NVDA currsor.
#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
2020.2
#### Windows version:
1909
#### Name and version of other software in use when reproducing the issue:
Windows update
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No
#### If addons are disabled, is your problem still occuring?
I don't think this is applicable, I don't have addons that effect currsor.
#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
No

**Comment by josephsl** [comment]
Hi, try with add-ons disabled and see if it is resolved; if yes, I know exactly what to do. Thanks.

**Comment by burakyuksek** [comment]
Hello,
No, the issue is not resolved by starting with add-ons disabled.

**Comment by josephsl** [comment]
Hi, hmmm, I see. If Narrator can announce this, chances are that it is reacting to a UIA event. Can you restart NVDA in debug logging mode (from Exit NVDA dialog with Windows 10 App Essentials installed) and try reproducing the issue, then copy and paste the log (or perhaps attach it)? That way we can figure out what’s up. Thanks.

**Comment by burakyuksek** [comment]
Hello,
Narrator can announce it if I move with capslock left/right. Does that change anything, should I still do what you requested?

**Comment by josephsl** [comment]
Hi, although that will explain a lot of your statement about Narrator, for sake of completeness, please do what I suggested. Thanks.

**Comment by burakyuksek** [comment]
[nvda-old.log](https://github.com/nvaccess/nvda/files/5092118/nvda-old.log)

**Comment by josephsl** [comment]
Hi, in short, NVDA is not seeing this text either because UIA is not giving us a complete picture or it is child of the first “learn more” button. Try pressing NVDA+Shift+Left arrow when you hear the first “learn more” button and see if NVDA notices this text. If it doesn’t, go back to “learn more” button and move through its child objects (if any) and see if the text in question is announced somehow. Thanks.

**Comment by burakyuksek** [comment]
Hello,
Pressing NVDA+shift+left arrow after hearing more info button didn't produce any results. Pressing NVDA+shift+down arrow on the button didn't either.

**Comment by josephsl** [comment]
Hi, another thing to try: what happens if you disable “simple review mode” found in NVDA’s review cursor panel? Thanks.

**Comment by burakyuksek** [comment]
Hello,
The announcement is presented when simple review is turned off.

**Comment by josephsl** [comment]
Hi, thought so. So it appears this is specific to NVDA. Once I locate a system (or a scenario) where I can reproduce this issue, I’ll think about possible workarounds from NVDA’s side (obviously it’ll go through the add-on first). Thanks.

**Comment by Adriani90** [comment]
@josephsl, @burakyuksek how is the experience now with Windows 10 22 H2 update?
I guess this is also very related to #13719. @josephsl is #13719 maybe even a duplicate of this?

**Comment by Adriani90** [comment]
@josephsl any feedback on my comment above, especially also with regard to #13719 please?

**Comment by josephsl** [comment]
Hi,

Since there will not be a new Windows 10 feature update and since NVDA can announce Windows 11 availability, I think it would be safe to close this issue (for now).
Thanks.
